cmd/estuary-shuttle/main.go

```diff
@@ -375,6 +375,7 @@ func main() {
 		})
 
 		go func() {
+			http.Handle("/debug/metrics/opencensus", estumetrics.Exporter())
```

Comment:
I believe this does the exact same thing as the next line

Reply:
Yes. I initially thought to separate an `exporter` and encapsulate it so that if in case we need to modify later we can just change it.